src/assets/thirdPartyServices/snet/example_service/index.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 
-import { Calculator } from './ExampleService/example_service_pb_service';
-import { getMethodNames } from '../../utility/sdk';
+import { Calculator } from './example_service_pb_service';
+import { getMethodNames } from '../../../../utility/sdk';
```

Comment:
lint: use double quotes

Reply:
🙈 Fixing it.

Reply:
Addressed in [commit](https://github.com/singnet/refactored-dapp/pull/118/commits/966dbdacbe6b3a4977252182578d69d918eb442c)